lib/src/test/java/io/ably/lib/test/rest/RestAuthAttributeTest.java

```diff
@@ -289,4 +290,80 @@ public Object getTokenRequest(TokenParams params) throws AblyException {
 			fail("auth_custom_options_authorise: Unexpected exception");
 		}
 	}
+
+	/**
+	 * Verify if {@link AuthOptions#force} is true
+	 * will to issue a new token even if an existing token exists.
+	 * <p>
+	 * Spec: RSA10d
+	 * </p>
+	 */
+	@Test
+	public void auth_authorise_force() {
+		try {
+			/* authorise with default options */
+			TokenDetails tokenDetails1 = ably.auth.authorise(null, null);
+
+			/* init custom AuthOptions with force value is true */
+			String custom_test_value = "test_forced_token";
+			AuthOptions authOptions = new AuthOptions() {{
+				authCallback = new TokenCallback() {
+					@Override
+					public Object getTokenRequest(TokenParams params) throws AblyException {
+						return custom_test_value;
+					}
+				};
+				force = true;
+			}};
+
+			/* authorise with custom AuthOptions */
+			TokenDetails tokenDetails2 = ably.auth.authorise(authOptions, null);
+
+			/* Verify that,
+			 * tokenDetails1 and tokenDetails2 aren't null,
+			 * tokens are different,
+			 * token from tokenDetails2 equals custom_test_value */
+			assertNotNull(tokenDetails1);
+			assertNotNull(tokenDetails2);
+			assertNotEquals(tokenDetails1.token, tokenDetails2.token);
+			assertEquals(tokenDetails2.token, custom_test_value);
+		} catch (AblyException e) {
+			e.printStackTrace();
+			fail("auth_custom_options_authorise: Unexpected exception");
+		}
+	}
+
+	/**
+	 * Verify if all AuthOption's attributes are null apart from force,
+	 * the previously configured authentication options will remain intact
+	 * (This behaviour takes precedence over RSA10j)
+	 * <p>
+	 * Spec: RSA10d
+	 * </p>
+	 */
+	@Test
+	public void auth_authorise_only_force() {
+		try {
+			/* authorise with default options */
+			TokenDetails tokenDetails1 = ably.auth.authorise(null, null);
+
+			/* init custom AuthOptions with provided force value only */
+			AuthOptions authOptions = new AuthOptions() {{
+				force = true;
+			}};
+
+			/* authorise with default options */
+			TokenDetails tokenDetails2 = ably.auth.authorise(authOptions, null);
+
+			/* Verify that,
+			 * tokenDetails1 and tokenDetails2 aren't null,
+			 * tokens are different */
+			assertNotNull(tokenDetails1);
+			assertNotNull(tokenDetails2);
+			assertNotEquals(tokenDetails1.token, tokenDetails2.token);
```

Comment:
I don't think you've asserted this correctly. You should set an attribute on the token request that is unique such as a `clientId` and then check again that the second token issued has the same attributes.  Otherwise "the previously configured authentication options will remain intact" is not being asserted which is the purpose of this test.